packages/boot/test/bootstrapTests/upgradeAPI.test.ts

```diff
@@ -0,0 +1,311 @@
+import { test as anyTest } from '@agoric/swingset-vat/tools/prepare-test-env-ava.js';
```

Comment:
it's a little hard to see the forest for the trees.

Part of me would like the uninteresting stuff moved to a separate file, but then I'd have to jump back and forth.

Reply:
Ah, of course.
(rhetorical)  Why didn't `yarn lint` warn me about an unused variable?
I can't easily add that to my pre-check script, since it's just a grep pattern.